wolfcrypt/test/test.c

```diff
@@ -389,7 +389,7 @@ static void myFipsCb(int ok, int err, const char* hash)
     #ifdef BENCH_EMBEDDED
         static byte gTestMemory[10000];
     #elif defined(USE_FAST_MATH) && !defined(ALT_ECC_SIZE)
-        static byte gTestMemory[130000];
+        static byte gTestMemory[140000];
```

Comment:
Was this increase required even in the !WOLFSSL_CERT_EXT case? If its just for the `WOLFSSL_CERT_EXT` case I'd like to only increase it for that build case.

Reply:
Got it